Dockerfile

```diff
@@ -12,8 +12,8 @@ WORKDIR /usr/src/app
 
 # First copy over the just the package.json files
 # so we can build a cached base image that only has node_modules
-COPY package.json package-lock.json ./
-COPY server/package.json server/package-lock.json server/
+COPY package.json ./
+COPY server/package.json server/
 RUN npm install && npm --prefix=server install
```

Comment:
If you keep the package-lock.json this could be replaced with `npm ci` instead of `npm install` and it would be a lot faster..

Reply:
[But totally fine if just deleting package-lock.json, which is what this PR does, so disregard.]